requirements/usage/use-cases.tex

```diff
@@ -0,0 +1,15 @@
+\subsection{Use cases}
+\begin{enumerate}
+  \item Add task.
+  \begin{itemize}
+    \item User inputs the task that has to be executed.
+    \item User adds data as well as constraints for the work machine to run the job on (both optional).
+    \item System verifies the user input.
+    If the there is an error, the user is informed and the use case ends.
+    \item System decides when and where to execute the task.
+    Based on preferences and availability.
+    \item System informs the User when the task is started.
+    \item User can check if the task is running correctly.

```

Comment:
`running correctly`? It seems strange in this context (makes sense if you take `User checks the state of the task.` scenario though